HIP/hip-xxx.md

```diff
@@ -0,0 +1,921 @@
+---
+hip: 801
+title: Add support for debugging EVM transactions
```

Comment:
I think we should be explicit here since there are many other debug API's that exist that aren't covered here but will be in the future
```suggestion
title: Add support for debug_traceTransaction RPC API
```